blueocean-material-icons/gulpfile.js

```diff
@@ -0,0 +1,80 @@
+"use strict";
+
+process.env.SKIP_BLUE_IMPORTS = 'YES';
+process.env.NODE_ENV = 'production';
+
+/*
+ Build file for BlueOcean Material Icons.
+ */
+
+const gulp = require('gulp');
+const fs = require('fs');
+const sourcemaps = require('gulp-sourcemaps');
+const babel = require('gulp-babel');
+const clean = require('gulp-clean');
+const runSequence = require('run-sequence');
+const lint = require('gulp-eslint');
+
+// Options, src/dest folders, etc
+
+const config = {
+    clean: ["dist", "target"],
+    react: {
+        sources: "src/**/*.{js,jsx}",
+        dest: "dist"
+    },
+    test: {
+        sources: 'test/js',
```

Comment:
So no tests are run, because you are storing them in the component. :bug:

Reply:
BTW why do you need a custom gulp?

Reply:
`js-builder` builds bundles which is not what we want if this is going to be consumed in non-Blue Ocean environments. Your point about cleaning up some of the references to test sources and possibly other items is valid and should be addressed. cc @NicuPascu